.gitignore

```diff
@@ -1,5 +1,4 @@
 # MegaLinter
```

Comment:
Commits: squash the two duplicate commits taking care to keep the second commit message, rather than the first and follow the 50/72 rule. We already removed `FAIL_IF_UPDATED_SOURCES=true` from our v5 hooks, so remove the final sentence.